.storybook/main.js

```diff
@@ -1,5 +1,7 @@
 const path = require("path");
 
+const isProduction = () => process.env.NODE_ENV === "production";
```

Comment:
Nie lepiej, żeby isProductuon było zmienną, a nie funkcja ? 🤔

Reply:
yup, w tym przypadku ma to sens - fixd in https://github.com/akai-org/EaterUI/pull/24/commits/61d4c3a57dc12848a05c7500dd76cc7d24c03009